client/components/foldable-card/index.jsx

```diff
@@ -56,13 +56,13 @@ class FoldableCard extends Component {
 		}
 	}
 
-	onClick = () => {
+	onClick = target => {
 		if ( this.props.children ) {
 			this.setState( { expanded: ! this.state.expanded } );
 		}
 
 		if ( this.props.onClick ) {
-			this.props.onClick();
+			this.props.onClick( target );
 		}
```

Comment:
it looks like we longer using `target` in our component. so we don't need to include this file in the changeset